src/psyclone/tests/psyir/nodes/assignment_test.py

```diff
@@ -133,28 +135,133 @@ def test_assignment_children_validation():
 
 def test_is_array_range():
     '''test that the is_array_range method behaves as expected, returning
-    true if the LHS of the assignment is an array range access and
-    false otherwise.
+    true if the LHS of the assignment is an array range access.
 
     '''
     one = Literal("1.0", REAL_TYPE)
     int_one = Literal("1", INTEGER_TYPE)
+
+    # lhs is an array reference with a range
+    array_type = ArrayType(REAL_TYPE, [10, 10])
+    symbol = DataSymbol("x", array_type)
+    my_range = Range.create(int_one, int_one.copy(), int_one.copy())
+    array_ref = ArrayReference.create(symbol, [my_range, int_one.copy()])
+    assignment = Assignment.create(array_ref, one.copy())
+    assert assignment.is_array_range
```

Comment:
Is this purposefully different to the rest i.e. no `is True`?

Reply:
Oops.